project/AppDependencies.scala

```diff
@@ -9,7 +9,8 @@ private object AppDependencies {
     ws,
     "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "3.4.0",
     "uk.gov.hmrc" %% "govuk-template" % "5.61.0-play-27",
-    "uk.gov.hmrc" %% "play-ui" % "8.21.0-play-27",
+    "uk.gov.hmrc" %% "play-frontend-govuk" % "0.60.0-play-27",
```

Comment:
Theres potential to update these possibly as showing available using sbt dependencyUpdates:
com.github.ghik:silencer-lib:provided                    : 1.7.0          -> 1.7.1                                            
[info]   com.github.ghik:silencer-plugin:plugin->default(compile) : 1.7.0          -> 1.7.1                                            
[info]   com.github.tomakehurst:wiremock-jre8:test,it             : 2.26.3                           -> 2.27.2                         
[info]   com.typesafe.play:play-ahc-ws                            : 2.7.5          -> 2.7.9          -> 2.8.7                          
[info]   com.typesafe.play:play-test:it,test                      : 2.7.5          -> 2.7.9          -> 2.8.7                          
[info]   org.mockito:mockito-core:it,test                         : 3.3.3                            -> 3.8.0                          
[info]   org.scala-lang:scala-library                             : 2.12.11        -> 2.12.13        -> 2.13.5                         
[info]   org.scalatestplus.play:scalatestplus-play:it,test        : 4.0.3                                              -> 5.1.0        
[info]   uk.gov.hmrc:auth-client                                  : 3.0.0-play-27                    -> 3.3.0-play-27  -> 5.2.0-play-28
[info]   uk.gov.hmrc:bootstrap-frontend-play-27                   : 3.4.0                                              -> 4.1.0        
[info]   uk.gov.hmrc:domain                                       : 5.10.0-play-27                   -> 5.11.0-play-27                 
[info]   uk.gov.hmrc:govuk-template                               : 5.61.0-play-27                   -> 5.65.0-play-27                 
[info]   uk.gov.hmrc:http-caching-client                          : 9.2.0-play-27                    -> 9.3.0-play-28                  
[info]   uk.gov.hmrc:play-frontend-govuk                          : 0.60.0-play-27                   -> 0.67.0-play-28                 
[info]   uk.gov.hmrc:play-frontend-hmrc                           : 0.50.0-play-27 -> 0.50.0-play-28 -> 0.54.0-play-28

Reply:
Updated what we can without breaking changes